Logger.Tests/LogFactoryTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+namespace Logger.Tests
+{
+    [TestClass]
+    public class LogFactoryTests
+    {
+        [TestMethod]
+        public void ConfigureFileLogger_FileLoggerNotInstantiated_ReturnsNull()
+        {
+            //Arrange
+            LogFactory logFactory = new LogFactory();
+            BaseLogger? logger;
+
+            //Act
+            logger = logFactory.CreateLogger("");
```

Comment:
```suggestion
            BaseLogger? logger = logFactory.CreateLogger(nameof(LogFactoryTests));
```

Be sure to use `nameof` when passing the class name to your logger.